custom/icds_reports/static/css/icds_dashboard.css

```diff
@@ -334,6 +334,10 @@ nvd3#sectorChart > div.title.h4 {
     color: #ffffff !important;
 }
 
+.report-content.icds-dashboard-title-fix {
```

Comment:
is this a thing we're not supposed to do? cc @orangejenny @biyeun 

(I wanted to avoid using `!important` and also ensuring I didn't break anything else so thought this was a way to ensure CSS took my rule)

Reply:
I guess I could also just define it last...

Reply:
I think you can safely ignore this, it looks like that rule exists for IE6 compatibility: https://github.com/CSSLint/csslint/wiki/Disallow-adjoining-classes

Reply:
ah, great thanks